kwctl-installer/action.yaml

```diff
@@ -7,7 +7,7 @@ inputs:
   KWCTL_VERSION:
     description: 'kwctl release to be installed'
     required: false
-    default: v1.6.0-rc2
+    default: v1.6.0-rc3
```

Comment:
As far as I can see, this will be updated to `v1.6.0-rc4`. Right?

Reply:
yes, I'll update the PR

Reply:
I'm using rc4 now